### ISSUE TITLE
Not working on product detail page

### ISSUE THREAD

**Issue by Anantkprajapati** [configure-tech/Configurator#3]
Hello Guys,

I noticed that this is not working on detail page and hide the fotorama gallery. 

This screenshot will help you.
https://prnt.sc/tz0ezq

Here is my local configuration
Theme - Luma
PHP - 7.2 centos & also checked on ubuntu
php version : 2.3.3

**Comment by kevin-f** [comment]
I'm not sure what you mean by it's not working on detail page; can you provide more detail?

As to the fotorama gallery, this is a known issue and appears to be caused by the fotorama plugin's integration. If you refresh the page, the gallery will usually appear on the second page load.

**Comment by Anantkprajapati** [comment]
Hello @kevin-f - can you please install this module on luma theme and check the product details page. it's effect to gallery and it's throwing error. When I am comment the code from 
https://github.com/configure-tech/Configurator/blob/master/view/frontend/layout/catalog_product_view.xml  https://prnt.sc/tzmeo8 and it's working fine.

**Comment by kevin-f** [comment]
Yes, but when you comment that code, the configurator will not instantiate. As I said yesterday, this is a known issue, and has nothing to do with our extension. As you can see from the screenshot you posted yesterday, the error comes from the fotorama extension, not from our extension. Also, if you refresh the page, the gallery should load fine.

**Comment by Anantkprajapati** [comment]
Hello @kevin-f - Yes, it's conflict issue with Fotorama but this is coming after adding this module. also it's loading first time but after refresh it's gone. so, if you can help with that then it will be good.

**Comment by kevin-f** [comment]
Hello @Anantkprajapati - I've spoken with our Magento developer, and it appears that this issue is not coming from the extension; rather, it appears to be coming from the configurator code itself. We are in the process of updating the configurator, and we will not be allocating programming resources to address this issue in the configurator code at this time, as it will be addressed in the update. Please bear with us until the update is released later this year.

**Comment by Anantkprajapati** [comment]
Okay thank you @kevin-f

**Comment by Anantkprajapati** [comment]
Hello @kevin-f  - is there any update? 
I still noticed same issue with latest version as well.

**Comment by kevin-f** [comment]
Hello, @Anantkprajapati - the configurator update that I was anticipating would be completed in the fall has been pushed back by other priorities, so, no; there have been no updates released as yet.

**Comment by Anantkprajapati** [comment]
@kevin-f  - is there any possible to send us any patch to fix this?

**Comment by kevin-f** [comment]
@Anantkprajapati - once again, the issue is not with the extension itself; it is with the configurator, which is an application hosted in the cloud. The extension only launches an instance of the configurator application into the requesting page on your store; any fixes to address this issue would have to be applied to the configurator application itself, not to instances of the extension.

**Comment by GJarosW2M** [comment]
@kevin-f  is there any update on this?  It's been almost two years and the client is still having this issue.

**Comment by kevin-f** [comment]
We've finished the front end rebuild of the configurator, and it is in testing now. It should be released shortly.
